app/models/filter.rb

```diff
@@ -102,7 +102,7 @@ def to_label
 
   def resource_type
     type = @resource_type || filterings.first.try(:permission).try(:resource_type)
-    type.presence
+    type || _('(Miscellaneous)')
```

Comment:
afaik you need to use `N_()` here for dynamic translations, and use `_()` when it been used in a erb/helper file for the actual translation process

Reply:
done, thanks